### PR TITLE
Sort taddr txns by txindex in addition to height

### PR DESCRIPTION
**Pull request by adityapk00**
When returning t-address txids, we need to sort by both height and txindex, so the transactions are returned in the same order they are in the blockchain.

### CODE REVIEW

File: src/rpc/misc.cpp [hunk 1]
```diff
@@ -1065,20 +1065,22 @@ UniValue getaddresstxids(const UniValue& params, bool fHelp)
     std::vector<std::pair<CAddressIndexKey, CAmount>> addressIndex;
     getAddressesInHeightRange(params, start, end, addresses, addressIndex);
 
-    // This is an ordered set, sorted by height, so result also sorted by height.
-    std::set<std::pair<int, std::string>> txids;
+    // This is an ordered set, sorted by (height,txindex) so result also sorted by height.
+    std::set<std::tuple<int, int, std::string>> txids;
```
Comment (str4d) [inline src/rpc/misc.cpp:1069]: Update the RPC documentation to say that from zcashd 4.5.0, txids are returned in the order they appear in blocks, which ensures that they are topologically sorted (i.e. parent txids will appear before child txids).

**Review by str4d — CHANGES_REQUESTED** [review]
LGTM.

File: src/rpc/misc.cpp (hunk 1 above)
Comment (CRYONE7) [reply src/rpc/misc.cpp:1069]: keys.txt/D1DB  F2C4  B96F  2DEB  F4C1  6651  0108  112E  7EA8  1F

Comment (adityapk00) [reply src/rpc/misc.cpp:1071]: Done, added the documentation

**Review by str4d — APPROVED** [review]
utACK

**Review by daira — COMMENTED** [review]
Post-hoc ACK.

**Pull request merged**